package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -368,6 +368,22 @@ define([
     }
   }, /** @lends pentaho.visual.base.View */{
 
+    /**
+     * Creates a view for the given model, of the registered default type, if any.
+     *
+     * If no default view type is registered
+     * for the given model type,
+     * or for any of its base model types,
+     * the returned promise is rejected.
+     *
+     * @param {!pentaho.visual.base.Model} model The visual model.
+     * @param {...any} other Other arguments to pass to the registered view constructor.
+     *
+     * @return {!Promise.<pentaho.visual.base.View>} A promise for a view of the given model.
+     *
+     * @rejects {pentaho.lang.ArgumentRequiredError} When `model` is not specified.
+     * @rejects {Error} When there is not registered default view class for the type of `model`.
```

Comment:
@rejects {Error} When there is not a registered default view class for the type of `model`.